common/mariadb/values.yaml

```diff
@@ -4,7 +4,7 @@
 # name: value
 
 name: null
-image: library/mariadb:10.6.21
+image: library/mariadb:10.6.22
```

Comment:
We should update version in the entrypoint script https://github.com/sapcc/helm-charts/blob/master/common/mariadb/scripts/docker-entrypoint.sh#L223

Reply:
fixed, by adding a regex lookup